orchestrator-salt/src/main/resources/salt/salt/sizes/init.sls

```diff
@@ -0,0 +1,19 @@
+/opt/salt/scripts/get_backup_data_sizes.sh:
+  file.managed:
+    - makedirs: True
+    - mode: 750
+    - source: salt://sizes/scripts/get_backup_data_sizes.sh
+
+/opt/salt/sizes/.pgpass:
+  file.managed:
+    - makedirs: True
+    - mode: 600
+    - source: salt://sizes/.pgpass
+    - template: jinja
+
+set_pgpass_file:
+  environ.setenv:
+    - name: PGPASSFILE
+    - value: /opt/salt/sizes/.pgpass
+    - require:
+      - file: /opt/salt/sizes/.pgpass
```

Comment:
The `get_backup_data_sizes.sh` is not used anywhere, what is the plan to run it and where?
Why would we need this if it is not executed for calculating backup sizes?

Reply:
https://github.com/hortonworks/cloudbreak/pull/13582/files#diff-1492601a5fe451d8dc1cb366b39d0222a6bcc00e1f75c75d1a69aee549bfe034R20